Source/com/Ids.h

```diff
@@ -54,7 +55,7 @@ namespace RPC {
         ID_SUBSYSTEM_LOCATION = 0x00000036,
         ID_SUBSYSTEM_IDENTIFIER = 0x00000037,
         ID_SUBSYSTEM_TIME = 0x00000038,
-		ID_SUBSYSTEM_SECURITY = 0x00000039,
+        ID_SUBSYSTEM_SECURITY = 0x00000039,
```

Comment:
Already exists as an interface in the ThunderInterfaces, no need to make this part of the Thunder.

Reply:
Sure, will use that from interfaces

Reply:
done